ultimatejobweb/migrations/0003_test_data_job.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations, transaction
+from ultimatejobweb.models import job, company
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ultimatejobweb', '0002_test_data_company'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        jobs = [('Production Engineer', 'https://www.facebook.com/careers/jobs/1672813472870915/'),
+                ('Product M', 'https://www.facebook.com/careers/jobs/16732472870915/')]
+
+        with transaction.atomic():
+            n = 0
+            for job_title, description_url in jobs:
+                n += 1
+                company_n = company.objects.get(id=n)
```

Comment:
I think you should have here an issue
you enter into  job(company=company_n, job_title=job_title, description_url=description_url)

How do you really know that the companies match?
You have 2 facebook jobs, and 2 companies Amazon and Facebook,
Can you explain

Reply:
exactly, I did that because I have 2 companies and 2 jobs

Reply:
Correct but look here both jobs are Facebook
And you are getting Compny(1) and Company(2)
While in reality it should be both jobs joined to single company,
Do you understand my claim here?

Reply:
The cleanest way to do this in one function is to put the company and jobs data in a single construct. That way you can save the company and insert its jobs (see below). In this case we have 2 jobs for facebook and 0 jobs for amazon:
CC: @omernu 

```
test_data = [('Facebook', 'https://www.facebook.com/careers/jobs?q=',
                      'templates/company logo/FBlogo.png', 'manipuletion_facebook', [('Production Engineer', 'https://www.facebook.com/careers/jobs/1672813472870915/'),
                ('Product M', 'https://www.facebook.com/careers/jobs/16732472870915/')]),
                     ('Amazon', 'https://www.amazon.jobs/en/search?base_query=&loc_query=',
                      'templates/company logo/AMAZONlogo.png', 'manipuletion_amazon',[])]

        with transaction.atomic():
            for company_name, company_search_url, company_logo, function_name, jobs in companies:
                company = Company(company_name=company_name, company_search_url=company_search_url, company_logo=company_logo,
                        function_name=function_name)
                company.save()
                for job_title, description_url in jobs:
                    Job(company=company, job_title=job_title, description_url=description_url).save()

```